### DISCUSSION TITLE
Drive - More Meta Information for Files and "Folders"

### DISCUSSION THREAD

**Discussion by tuefekci** [deta/cloud-docs#265] [Ideas]
Hello, I don't know if there are plans already somewhere, so if this is a duplicate, just close it.

So to the main topic, I would love for drive to have more meta information available especially date but the more, the better if it does not complicate everything too much.

Why would this be helpfully I depend on how you use drive, but I find it hard to clean it for example, so I often tend to basically remove everything of course you could work with meta tags in the name field, but I personally find that makes it more complicated everywhere else, so **my favorite would be to add file date and file size at least add functionality separately to get the meta information.**

Of course, you could separately write everything into base as well which I probably will pivot to for the moment, but this is also unreliable if something goes wrong, or I will place meta files beside the file but stop gate solutions are stupid, and I think the backend should have file sizes and dates already even if the files are in a database there should still be simple ways to obtain these values.

I would love for this to be implemented, but I will not be angry if there is really something standing in the way of it, or it would use a lot of  resources, but at least from my viewport it looks more like an oversight than a technical problem.

Also, a big thanks to Deta for saving us all costs while we develop and test stuff!

**Comment by jnsougata** [comment]
Yes deta drive supports folder. If you want to add a file to a folder just add the folder name followed by `/` at the start of the file name.
Exmaple: your file name is `test.png` and folder name is `images`. what you can do is set file name to `images/test.png` while uploading to drive and it will add `images` folder to the drive it the folder doesn't exist else it will just add the file `test.png` inside the folder.

**Reply by tuefekci** [reply]
Sorry my headline is a bit misleading, I meant add meta information for  folders not add folders.

**Reply by jnsougata** [reply]
...

**Comment by abdelhai** [comment]
hi, thanks for the feedback! we will take note of this for our next release. seems useful.

using base sound like a good "work-around".
